core/store/migrations/0043_add_uuid_to_pipeline_task_runs.go

```diff
@@ -0,0 +1,77 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up43 = `
+	-- Truncate tables to ease bigint -> UUID migration
+	TRUNCATE TABLE pipeline_runs, pipeline_task_runs, flux_monitor_round_stats_v2;
```

Comment:
ufff, we're going to need to warn people about this one. Was it not practical to assign UUIDs to existing data?

Reply:
There can be a large amount of existing runs, @tyrion70 said it's probably fine to truncate (if I avoid cascading)

Reply:
Oh we'll definitely need to add something to the CHANGELOG about this, can be done in a follow up